projects/sbb-esta/angular-public/breadcrumb/src/styles/_breadcrumb-base.scss

```diff
@@ -75,21 +85,35 @@
     display: flex;
   }
 
+  &:first-of-type a {
```

Comment:
this block should be business only (see messed up public styling)

Reply:
I did make these two blocks business only, but I don't understand exactly what you mean by "messed up public styling". If you mean the misaligned text and icons that seemed to already be an issue before my changes [see here](https://angular-staging.app.sbb.ch/latest/public/components/breadcrumb). Obviously I can try to fix it but I'm not sure this PR is the right place to do that :)

Reply:
Sorry for the unclear comment. The problem was, that the last entry was displayed in another color. compare: https://angular.app.sbb.ch/latest/public/components/breadcrumb
with https://angular-staging.app.sbb.ch/353/public/components/breadcrumb (but maybe in PR it's already fixed :) )

Reply:
The misalignment in breadcrumb is definitely a problem which has to be fixed. Another PR would be the best solution :). Do you like to fix that for us?

Reply:
I can definitely do that. Shall I create a new issue for it or just submit the PR once I'm done?

Reply:
Only a PR will be ok. Thank you!